File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/05 - Conditional Statements Exercise/01. Sum Seconds .py
time_first = int(input())
time_second = int(input())
time_third = int(input())

total_time = time_first + time_second + time_third

minutes = total_time // 60
seconds = total_time % 60

# това е единият начин за принтиране на секунди чрез форматиране за добавяне на 0та :)
# print(f"{minutes}:{seconds:02d}")

if seconds < 10:
    print(f"{minutes}:0{seconds}")
else:
    print(f"{minutes}:{seconds}")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/08. УПРАЖНЕНИЕ ПРОВЕРКИ/01. Sum Seconds.js
function sumSeconds(input) {
    let totalTime = Number(input.shift()) + Number(input.shift()) + Number(input.shift());
    
    let minutes = ~~(totalTime / 60);
    let seconds = totalTime % 60;
    
    if (seconds < 10) {
        console.log(`${minutes}:0${seconds}`);
    } else {
        console.log(`${minutes}:${seconds}`);
    }
}

sumSeconds(["35", "45", "44"]);  // Expexted output: 2:04
sumSeconds(["22", "7", "34"]);  // Expexted output: 1:03
sumSeconds(["50", "50", "49"]);  // Expexted output: 2:29
sumSeconds(["14", "12", "10"]);  // Expexted output: 0:36
